tests/pointings_search/test_pointings_table.py

```diff
@@ -143,6 +143,54 @@ def test_to_csv_cached():
         assert "earth_vec_z" in data2.pointings.columns
 
 
+def test_to_sqlite():
+    """Confirm that we can save and reload the basic data."""
+    data_dict = {
+        "ra": [0.0, 90.0, 45.0, 90.0, 270.0],
+        "dec": [0.0, 90.0, 0.0, 45.0, 0.0],
+        "obstime": [0.0, 1.0, 2.0, 3.0, 4.0],
+        "flux": [10.0, 10.0, 10.0, 10.0, 10.0],
+    }
+    data = PointingTable.from_dict(data_dict)
+    data.preprocess_pointing_info()
+    data.append_earth_pos()
+
+    with tempfile.TemporaryDirectory() as dir_name:
+        filename = path.join(dir_name, "tmp_test.db")
+        data.to_sqlite(filename, "pointings")
+
+        # Check that we can reload it. Use all columns.
+        data2 = PointingTable.from_sqlite(filename, "pointings")
+        assert len(data2.pointings) == 5
+        assert len(data2.pointings.columns) == 10
+        assert np.allclose(data2.pointings["ra"], data_dict["ra"])
+        assert np.allclose(data2.pointings["dec"], data_dict["dec"])
+        assert np.allclose(data2.pointings["obstime"], data_dict["obstime"])
+        assert np.allclose(data2.pointings["flux"], data_dict["flux"])
+        assert "unit_vec_x" in data2.pointings.columns
+        assert "unit_vec_y" in data2.pointings.columns
+        assert "unit_vec_z" in data2.pointings.columns
+        assert "earth_vec_x" in data2.pointings.columns
+        assert "earth_vec_y" in data2.pointings.columns
+        assert "earth_vec_z" in data2.pointings.columns
+
+        # Load only a subset of the data.
+        colmaps = {"obstime": "obstime", "ra": "ra", "dec": "dec"}
```

Comment:
Can you add a test that uses different names for the columns?

Reply:
Added.